scripts/create-deploy-pr

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+ORIGIN=origin
+DEPLOY_BRANCH=stages/prod
+
+if [ "${CI:-}" == "" ]; then
+    echo "This script is meant to be run in a continuous integration environment."
+    exit 1
+fi
+
+SHA=$(curl --silent https://idp.staging.login.gov/api/deploy.json | jq -r .git_sha)
+echo "Preparing deploy PR for ${SHA}"
+
+LAST_RC=$(gh release list --exclude-drafts --exclude-pre-releases --limit 1 | grep -E --only-matching 'RC [0-9]+(\.[0-9]+)?' | sed 's/RC //')
+if [ "$LAST_RC" == "" ]; then
+    LAST_RC=0
+fi
+
+# TODO: Patch support
+LAST_RC_MAJOR=$(bc <<< "scale=0; ${LAST_RC} / 1")
+NEXT_RC=$((LAST_RC_MAJOR + 1))
+echo "Last RC was ${LAST_RC}. The next RC will be ${NEXT_RC}."
+
+RC_BRANCH=stages/rc-$(date +'%Y-%m-%d')
+
+if git rev-parse $RC_BRANCH > /dev/null 2>&1; then
+    echo "RC branch $RC_BRANCH already exists."
+    exit 1
+fi
+
+git checkout -b $RC_BRANCH
+git reset --hard $SHA
+
+scripts/changelog_check.rb -b "${ORIGIN}/${DEPLOY_BRANCH}" > rc_changelog.md
+
+git push -u $ORIGIN HEAD
+
+if [ "${DRY_RUN:-}" == "" ]; then
+    # Create PR
```

Comment:
Creating the PR and draft release don't seem destructive to me. Do we need the DRY_RUN flag?

Reply:
Creating the PR/Draft release are visible changes, and make API requests. If I were testing this script locally, I'd want an option to not create them and potentially spam the rest of the team (I get emails for every PR & release created, others prob do too)

Reply:
I made a few tweaks and added a `--dry-run` arg, mostly to support my own local testing